### PR TITLE
Test on kenny

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

**Comment by ipa-nhg** [comment]
@bajo @mzillich please review
@lokalmatador I needed to change the namespaces to integrate the arm driver, I hope it is ok https://github.com/ipa-nhg/squirrel_driver/blob/43b107cba848a063ea180ec6c94ca04bc86bf2f2/squirrel_control/src/squirrel_hw_interface.cpp#L41

**Comment by ghost** [comment]
what What WHAAAAAAAAAT?!?!?!

Nah, just joking. For the sake of having a working robot you're free to do whatever you want :)

**Comment by ipa-nhg** [comment]
@lokalmatador  :smile: thanks

if you run robotino_bringup robot.launch with the new version all the topics and parameters for the arm will be mapped to `/arm_controller/...` and I you have a relay for : 
```
/arm_controller/robot_description -> /robot_description
/arm_controller/joint_states -> /joint_states
```
I tested and is working, also the tf tree look good

File: squirrel_interaction/src/squirrel_interaction/board/serial_api.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
 
 The board controls the following devices:
 
-    - Head Motor (pan) (-180º ~ 180ª)
+    - Head Motor (pan) (-180º ~ 180ª) - head_controller
```
Comment (bajo) [inline squirrel_interaction/src/squirrel_interaction/board/serial_api.py]: It is only a comment but can you please also change the wrong letters ª to ° while we are already changing this file.
I think this happened 3 times. Thank you.

**Comment by bajo** [comment]
Other than the small comment on the comments this PR looks good to me.

**Comment by ipa-nhg** [comment]
Done: https://github.com/squirrel-project/squirrel_driver/pull/138/commits/6d209ed6463efb026931b10d9a68ffcdea56469d

**Comment by bajo** [comment]
Thanks.
LGTM

**Pull request merged**